src/System.Linq/samples/AggregateSamples.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System;
+
+namespace System.Linq.Samples
+{
+    public static class AggregateSamples
+    {
+        /// <summary>
+        /// Using Aggregate to reverse a string.
+        /// </summary>
+        public static void Overload1()
+        {
+            string sentence = "there's nothing quite like ice cream";
+
+            string[] words = sentence.Split(' ');
+
+            string reversedSentence = words.Aggregate((workingSentence, nextWord) => nextWord + " " + workingSentence);
+
+            // reversedSentence => "cream ice like quite nothing there's"
+
+            Console.WriteLine(reversedSentence);
+        }
+
+        /// <summary>
+        /// Using Aggregate to sum the lengths of the words in a string.
+        /// </summary>
+        public static void Overload2()
+        {
+            string sentence = "there's nothing quite like ice cream";
+
+            string[] words = sentence.Split(' ');
+
+            int seed = 0;
+            int sumOfWords = words.Aggregate(seed, (workingTotal, nextWord) => workingTotal + nextWord.Length);
+
+            // sumOfWords => 31
+
+            Console.WriteLine($"Sentence: \"{sentence}\"\nHas {sumOfWords} total non-whitepsace characters!");
```

Comment:
I think the solution file is fine. It has a separate "MinimumVisualStudioVersion" tag, which looks like it should still be fine. I think most of our solution files in here say that the "VisualStudioVersion" is 14.0, but they should still load in 12.0 as long as the min version is less than that.